.github/workflows/publish.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish Release to NuGet
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: 8.x.x
+
+    - name: Build SDK
+      run: dotnet build
+
+    - name: Publish to NuGet
```

Comment:
This will fail if we don't do a tag check.  i.e. we might want to trigger only on tag create?

Reply:
btw... I could go either way on this as we iterate.

Reply:
I've copied this from the [Octokit.net publish YAML](https://github.com/octokit/octokit.net/blob/main/.github/workflows/publish.yml), and it looks like it [does only trigger on tag creation](https://github.com/octokit/dotnet-sdk/pull/12/files/29adf0b45351939011c3d5e0179761e20afbca07#diff-551d1fcf87f78cc3bc18a7b332a4dc5d8773a512062df881c5aba28a6f5c48d7R3-R6). Am I missing something here?

Reply:
🤦 I just now saw the on push tags instruction... not quite sure how I missed that... sorry for the noise.

Reply:
No problem!